supervisord.conf

```diff
@@ -8,7 +8,7 @@ port=:9001
 supervisor.rpcinterface_factory = supervisor.rpcinterface:make_main_rpcinterface
 
 [program:jenkins]
-command=/usr/local/bin/jenkins.sh
+command=/usr/local/bin/shim-startup-wrapper.sh
```

Comment:
@rtyler WDYT of this as a temporary measure to be able to move forward? 

At least it's very useful for me locally to be able to iterate.